cmd/ris/api/ris.proto

```diff
@@ -58,6 +65,7 @@ message ObserveRIBRequest {
     }
     AFISAFI afisafi = 3;
     bool allow_unready_rib = 5;
+    optional ObserveRIBOptions observe_options = 6; // additional options for the observe request
```

Comment:
Is there a reason you're just not extending the `ObserveRIBRequest` instead of defining another message?